I am getting an error within the catch.hpp file however I think it may be in my test.cpp file not sure though.